app/models/user.rb

```diff
@@ -610,7 +700,7 @@ def former_passwords_include?(password)
   def clean_up_former_passwords
     # minimum 1 to keep the actual user password
     keep_count = [1, Setting[:password_count_former_banned].to_i].max
-    (passwords[keep_count..-1] || []).each(&:destroy)
+    (passwords[keep_count..] || []).each(&:destroy)
```

Comment:
:green_circle: Probably that hint comes too late now, but this file contains a bunch of rubocop-induced code changes that would have probably belonged into a separate commit (maybe even PR, but with a separate commit that would've been a decision for later :) )

Anyways:

:heart: for tackling all of those rubocop offenses :)

Reply:
I used `rubocop -a` to format this file and decided to squeeze additional fixes in. True, they are irrelevant to the PR. Later I will separate them at least by having a dedicated commit. Accept my apologies for overwhelming :pray: